.travis/run_travis_job.sh

```diff
@@ -53,4 +53,25 @@ else
     if [[ "${TRAVIS_JOB_NAME}" == "Linux JDK 8 CHECK_RUST=false" ]]; then
       mvn org.eluder.coveralls:coveralls-maven-plugin:report
     fi
+
+    # Test exonum_launcher_java_plugins
+    if [[ "${TRAVIS_JOB_NAME}" == "Linux JDK 8 CHECK_RUST=false" ]]; then
+      cd "${TRAVIS_BUILD_DIR}/exonum-java-binding/exonum_launcher_java_plugins"
+      # Install pip
+      curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+      python3.7 get-pip.py --user
+      # Install exonum-launcher
+      git clone https://github.com/exonum/exonum-launcher.git
+      pip3 install --user -r exonum-launcher/requirements.txt
+      pip3 install --user -e exonum-launcher --no-binary=protobuf protobuf
+      # Install exonum_launcher_java_plugins
+      pip3 install --user -e .
+      # Download latest protobuf compiler
+      wget https://github.com/protocolbuffers/protobuf/releases/download/v3.11.3/protoc-3.11.3-linux-x86_64.zip
+      unzip protoc-3.11.3-linux-x86_64.zip
+      export PROTOC="$(pwd)/bin/protoc"
```

Comment:
Shan't we add protoc to `PATH`?

Reply:
It was my initial attempt, but actually exonum-python-client is looking for `PROTOC` env var, so it's enough.